src/react-native/02-setting-up-your-environment/setting-up-your-environment.md

```diff
@@ -511,7 +511,7 @@ C:\Users\bitovi\AppData\Local\Android\Sdk\platform-tools
 
 ### Objective 6: Launch the Android emulator
 
-<img alt="Screenshot of the Android Emulator running in Android Studio" src="../static/img/react-native/02-setting-up-your-environment/06-solution.png" style="max-height: 640px; border: 4px solid black; border-radius: 25px;"/>
+<img alt="Screenshot of the Android Emulator running in Android Studio" src="../static/img/react-native/02-setting-up-your-environment/06-solution.png" style="border: 4px solid black; border-radius: 25px;" width="640"/>
```

Comment:
These had a `max-height` so I think the `height` attribute should be set instead.